src/main/java/org/apache/commons/codec/language/MatchRatingApproachEncoder.java

```diff
@@ -126,10 +126,20 @@ public final String encode(String name) {
         // Preprocessing
         name = cleanName(name);
 
+        // Bulletproof if name becomes empty after cleanName(name)
+        if (EMPTY.equalsIgnoreCase(name) || SPACE.equalsIgnoreCase(name) || name.length() == 0) {
```

Comment:
Use `isEmpty()`.

Reply:
- You're checking that the name is empty... twice?
- Why use equalsIgnoreCase() for SPACE?